fix(docs): correct resource IDs for importing delegated permissions and claims mapping policy (#1648)
hashicorp/terraform-provider-azuread: docs/resources/claims_mapping_policy.md

## Import

> Claims Mapping Policy can be imported using the object ID of the associated service principal and the ID of the claims mapping policy, e.g.

Reviewer: This import description looks like it may have been meant for the `azuread_service_principal_claims_mapping_policy_assignment` resource below?

```suggestion
Claims Mapping Policies can be imported using the `id`, e.g.
```

```shell

> terraform import azuread_claims_mapping_policy.my_policy /servicePrincipals/00000000-abcd-0000-0000-000000000000/claimsMappingPolicies/00000000-efgh-0000-0000-000000000000

Reviewer 2: I think this should be:
```suggestion
terraform import azuread_claims_mapping_policy.my_policy /policies/claimsMappingPolicies/00000000-0000-0000-0000-000000000000
```
This is a different resource/ID  to `azuread_service_principal_claims_mapping_policy_assignment`
